pcsx2/Memory.h

```diff
@@ -48,6 +48,7 @@ static __fi void ZeroQWC( u128& dest )
 #define psHs16(mem)	(*(s16*)&eeHw[(mem) & 0xffff])
 #define psHs32(mem)	(*(s32*)&eeHw[(mem) & 0xffff])
 #define psHs64(mem)	(*(s64*)&eeHw[(mem) & 0xffff])
+#define psHs128(mem)(*(s128*)&eeHw[(mem) & 0xffff])
```

Comment:
Tab after (mem), should match the rest of the code, same goes for the other new lines that are added.